source/guide_grocy.rst

```diff
@@ -0,0 +1,144 @@
+.. highlight:: console
+
+.. author:: Jonas <https://github.com/jfowl>
+
+.. tag:: lang-php
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/grocy.svg
+      :align: center
+
+####
+Grocy
+####
+
+.. tag_list::
+
+Grocy_ is a web-based self-hosted groceries & household management solution for your home.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Domains <web-domains>`
+
+License
+=======
+
+Grocy is released under the `MIT license`_.
+
+Prerequisites
+=============
+
+Your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Step 1
+-----
+
+Download grocy to your home directory.
```

Comment:
Installing php code into the home directory will not work because the web server is only allowed to access files in `/var/www/virtual/...`:

```
[Mon Aug 31 13:29:44 2020] [error] [pid 18613] request.c(1204): [client 2a02:8388::] AH00037: Symbolic link not allowed or link target not accessible: /var/www/virtual/grocy2/html
```

Did you actually test this? :sweat_smile: 

You can install the files into `/var/www/virtual/$USER/grocy` instead.

Reply:
Before I tested it I mentioned it as an option in my review but then I ran into exactly that issue :laughing: - So I removed that passage quickly. I should have mentioned that. 
This is a working structure like mentioned before:
```
/var/www/virtual/isabell/
├── grocy
├── someStaticFiles.example.com
└── html -> grocy/public
```

re-add this line before doing the magic and everything should work:
```
[isabell@stardust ~]$ cd /var/www/virtual/$USER/
```